provisioning/device/src/Transports/Amqp/ProvisioningTransportHandlerAmqp.cs

```diff
@@ -335,7 +334,7 @@ private void ValidateOutcome(Outcome outcome)
                     }
 
                     throw new ProvisioningClientException(
-                        rejected.Error.Description,
```

Comment:
This would use the full content of error description object as the Message property of `ProvisioningClientException` thrown here but we don't need information more than the error message itself appearing in this property, so I am correcting it here.